### PR TITLE
Rename stuff to Element

### PR DESCRIPTION
**Pull request by lieuwex**

### CODE REVIEW

File: gatsby/content/faq/encryption.mdx [hunk 1]
```diff
@@ -15,12 +15,12 @@ End-to-End Encryption is currently in late beta. Rooms can have encryption enabl
 
 End-to-end encryption is currently available in:
 
-- [Riot Web](https://github.com/vector-im/riot-web/)
+- [Element Web](https://github.com/vector-im/element-web/)
   - and [matrix-js-sdk](https://github.com/matrix-org/matrix-js-sdk)
   - and [matrix-react-sdk](https://github.com/matrix-org/matrix-react-sdk)
-- [Riot Android](https://github.com/vector-im/riot-android/)
+- [Element Android](https://github.com/vector-im/element-android/)
   - and <a href="https://github.com/matrix-org/matrix-android-sdk">matrix-android-sdk</a>
```
Comment (tulir) [inline gatsby/content/faq/encryption.mdx:22]:
```suggestion
  - and <a href="https://github.com/matrix-org/matrix-android-sdk2">matrix-android-sdk</a>
```

**Comment by tulir** [comment]
btw the FAQ seems to have a bunch of other places where it still says Riot

File: gatsby/content/faq/encryption.mdx (hunk 1 above)
Comment (lieuwex) [reply gatsby/content/faq/encryption.mdx:22]: Thanks. Seems like the name is changed too, looking at this: https://github.com/matrix-org/matrix-android-sdk2#how-to-integrate-the-sdk-in-your-application

**Comment by lieuwex** [comment]
Hmm indeed, I just saw the stuff under the encryption header being wrong. So I searched for that.

https://github.com/matrix-org/matrix.org/pull/818/commits/e178064ff87f046761323323ea3dc775229e8323 is the rest of the FAQ done I hope. (only not the Russian part, since I don't want to touch that without knowledge of Russian)

**Review by benparsons — APPROVED** [review]
lgtm

**Pull request merged**